SDC/index.js

```diff
@@ -1,46 +1,96 @@
-const { Client } = require( 'pg' );
-const express = require( 'express' );
-let app = express();
-
+require('newrelic');
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const dotenv = require('dotenv');
+const Review = require('./Review.js');
+const pool = require( './pgsql.js' );
+// const { Pool } = require( 'pg' );
+// const cass = require( './cassie.js' );
+// const Sequelize = require('sequelize');
```

Comment:
Eventually you'll want to clean out commented lines of code. If you are using Postgres, you can get rid of Cassandra lines, right? If you ever really need to switch you could add it back in, or an older commit will have it.